versions.json

```diff
@@ -0,0 +1,25 @@
+{
+    "CoreData Desktop": {
+	"version": "3.0.1",
+	"important": true,
+	"minVersion": "2.5.1",
+	"binary": "http://coredata.is/setup/coredata_desktop-${ARCH}-3.0.1.0.msi"
```

Comment:
Yes, no doubt. This was originally just a demo file, copied from the server. I'll update the one on the server and this showcase version.